### PR TITLE
Added jceks support. Uses jceks package to parse files.

### PR DESCRIPTION
**Pull request by christodenny**
@csstaub @alokmenghrajani @mcpherrinm @worldwise001 ptal

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -115,6 +116,21 @@ func getCerts(file, format string) ([]*x509.Certificate, error) {
 				certs = append(certs, cert)
 			}
 		}
+	case "JKS":
```
Comment (csstaub) [inline main.go]: You mean JCEKS. JKS is a different format.

Comment (christodenny) [inline main.go]: changed, @csstaub

File: main.go [hunk 2]
```diff
@@ -115,6 +116,21 @@ func getCerts(file, format string) ([]*x509.Certificate, error) {
...
+		scanner := bufio.NewReader(os.Stdin)
+		fmt.Print("Enter password: ")
+		password, _ := scanner.ReadString('\n')
+		keyStore, err := jceks.Load(file, []byte(strings.TrimSuffix(password, "\n")))
+		if err != nil {
+			return nil, err
+		}
+		for _, certName := range keyStore.ListCerts() {
```
Comment (csstaub) [inline main.go]: You should also iterate over private key entries, as they have certificates associated with them.

Comment (christodenny) [inline main.go]: added in latest commit @csstaub

File: main.go [hunk 3]
```diff
@@ -115,6 +116,30 @@ func getCerts(file, format string) ([]*x509.Certificate, error) {
...
+		}
+		for _, alias := range keyStore.ListCerts() {
+			cert, _ := keyStore.GetCert(alias)
+			if err != nil {
+				return nil, err
+			}
+			certs = append(certs, cert)
+		}
+		for _, alias := range keyStore.ListPrivateKeys() {
+			fmt.Print("Enter password for Alias [" + alias + "]: ")
```
Comment (csstaub) [inline main.go]: Use fmt.Printf here

Comment (csstaub) [inline main.go]: Also, "Alias" should probably be "alias"

Comment (christodenny) [inline main.go]: updated @csstaub

**Comment by csstaub** [comment]
👍

**Pull request merged**